File: Python Exams/Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/05. Care of Puppy/05. Care of Puppy.py
food = int(input()) * 1000
eaten_food = 0

while True:
    gr_food = input()

    if gr_food == 'Adopted':
        break
    else:
      eaten_food += int(gr_food)

if food - eaten_food >= 0:
    print(f'Food is enough! Leftovers: {food - eaten_food} grams.')     
else:
    print(f'Food is not enough. You need {abs(food - eaten_food)} grams more.')  

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/05. Care of Puppy/05. Care of Puppy.js
function careOfPuppy(arg) {
let food = Number(arg.shift()) * 1000
let removeAdopted = arg.pop()
let eaten_food = arg.reduce((a, b) => +a + +b)

if (food - eaten_food >= 0)
    return `Food is enough! Leftovers: ${food - eaten_food} grams.`
return `Food is not enough. You need ${eaten_food - food} grams more.`
}

// console.log(careOfPuppy([
// 4,
// 130,
// 345,
// 400,
// 180,
// 230,
// 120,
// 'Adopted',
// ]));